tensorflow/lite/micro/kernels/mul.h

```diff
@@ -41,11 +41,25 @@ struct OpDataMul {
   float output_activation_max_f32;
 };
 
+void* MulInit(TfLiteContext* context, const char* buffer, size_t length);
+
 TfLiteStatus CalculateOpDataMul(TfLiteContext* context, TfLiteNode* node,
                                 TfLiteMulParams* params, OpDataMul* data);
 
 TfLiteStatus MulPrepare(TfLiteContext* context, TfLiteNode* node);
 
+void EvalQuantizedReference(TfLiteContext* context, TfLiteNode* node,
+                            const OpDataMul* data,
+                            const TfLiteEvalTensor* input1,
+                            const TfLiteEvalTensor* input2,
+                            TfLiteEvalTensor* output);
+
+void EvalFloatReference(TfLiteContext* context, TfLiteNode* node,
+                        TfLiteMulParams* params, const OpDataMul* data,
+                        const TfLiteEvalTensor* input1,
+                        const TfLiteEvalTensor* input2,
+                        TfLiteEvalTensor* output);
```

Comment:
Since these are no longer in an anonymous namespace, let's add Mul to the function names.

perhaps ```EvalMulReferenceFloat``` and ```EvalMulReferenceQuantized```?

Reply:
Done.